pom.xml

```diff
@@ -51,8 +51,8 @@
         <slf4j.version>1.7.25</slf4j.version>
         <slf4j-log4j12.version>1.7.25</slf4j-log4j12.version>
         <guava.version>19.0</guava.version>
-        <netty.version>4.0.56.Final</netty.version>
-        <netty-tcnative.version>2.0.7.Final</netty-tcnative.version>
+        <netty.version>4.1.75.Final</netty.version>
+        <netty-tcnative.version>2.0.50.Final</netty-tcnative.version>
```

Comment:
Upgraded to match [what's in the 4.1.75 pom.xml](https://github.com/netty/netty/blob/netty-4.1.75.Final/pom.xml#L545) based on [this suggestion](https://github.com/netty/netty-tcnative/issues/366) from Norman

Reply:
Excellent, this is indeed a common source of issues when upgrading Netty.